apps/src/code-studio/components/progress/StatusProgressDot.jsx

```diff
@@ -1,11 +1,15 @@
+// TODO: rename file now that it wraps ProgressBubble instead of dot
```

Comment:
I should probably make the TODO more clear, but my intent was to do this once this isnt behind an experiment any longer